pom.xml

```diff
@@ -15,61 +15,69 @@
   ~ limitations under the License.
   -->
 
-<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>de.codecentric</groupId>
-  <artifactId>chaos-monkey-spring-boot-parent</artifactId>
-  <version>${revision}</version>
-  <packaging>pom</packaging>
-  <url>https://github.com/codecentric/chaos-monkey-spring-boot</url>
-  <name>chaos-monkey-spring-boot-parent</name>
-  <description>Chaos Monkey for Spring Boot parent project</description>
-
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
   <properties>
     <!-- project version -->
-    <revision>2.3.0-SNAPSHOT</revision>
+    <asciidoctor-maven-plugin.version>1.6.0</asciidoctor-maven-plugin.version>
```

Comment:
Was the revision deleted on purpose? And if so the comment would be wrong.

Else changes looks good and could be merged. 👍  I‘ll aprove it but question frim above should ve answered before we merge it.

Reply:
The revision should only have been moved because of the formatting. Can you double check its there? I think it is now in: Line 61

Reply:
@WtfJoke ping

Reply:
Yes youre right, its there. Sorry didnt have time earlier. 😞